CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 ## What's New
 
 * CLI additions for `ziti` to login with certificates or external-jwt-signers
+* NOTE: ziti edge login flag changes:
+  * `-c` flag has been changed to map to `--client-cert`
```

Comment:
It's difficult to estimate how much user pain we can expect from this change. I'm sure there will be some, but the new options are more clear. `--cert` was pretty vague, so that's an improvement.

I suggest that we have a uniform long option name, e.g. `--ca-cert` for all this CLI's sub-commands, or make it a global option. For example, `--ca-cert` is  already used by this CLI. The uniformity seems more important than reverse compatibility.